lib/deployer/notificationTypes.js

```diff
@@ -1,9 +1,9 @@
 const { executeHttpRequest } = require("@sap-cloud-sdk/http-client");
 const { buildHeadersForDestination } = require("@sap-cloud-sdk/connectivity");
+const { createLogger } = require("@sap-cloud-sdk/util");
+const LOG = createLogger('notifications');
```

Comment:
what is the advantage of using `@sap-cloud-sdk/util` over `cds.log`. IMO `cds.log` should cover all the required features and if not those should be requested

Reply:
The intention was that the deployer module runs with the bare minimum required dependencies and utils is anyways needed for executeHttpRequest thus @sao/cds as a dependency for the deployer can be avoided.